### PR TITLE
ROC-3153: Delivery pipeline functional tests (debugging)

### PR DESCRIPTION
**Pull request by rlewan**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ROC-3153

### Change description ###

Enables a possibility to execute selected integration tests against an AAT instance in CNP pipeline.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -161,6 +164,15 @@ task apiTest(type: Test) {
   setClasspath(sourceSets.integrationTest.runtimeClasspath)
 }
 
+task functional(type: Test) {
```
Comment (yarpo) [inline build.gradle]: `functionalTest` wouldn't be a better name for this task. Consistent with `apiTest`

Comment (timja-kainos) [reply build.gradle]: name that is set by the CNP team, we're just implementing the interface

Comment (rlewan) [reply build.gradle]: That's not my decision -- that's the task CNP pipeline is expecting to call.

Comment (yarpo) [reply build.gradle]: ok

**Comment by rlewan** [comment]
Closing this, will change approach to a separate source set for dedicated functional tests.

**Pull request closed without merging**